services/pdf-ms/lambdas/addPdfToCase.ts

```diff
@@ -26,7 +26,7 @@ export const main = async (event: Record<string, any>) => {
   if (user) caseData.answers.user = user;
 
   // find changed and newly added values, compared to the last case with the same formId.
-  const relevantCases = await getRelevantCases(personalNumber, formId);
+  const relevantCases = await getCasesWithFormId(personalNumber, formId);
```

Comment:
do i explicit need to know it's relevantCases. Why not just cases?

Reply:
The idea was to signal that it's not just any or all cases, but some specific ones.